src/server.js

```diff
@@ -0,0 +1,40 @@
+const FeedParser = require('feedparser')
+const request = require('request')
+const Rx = require('rx') 
+
+// Event loop will run 3 times for testing
+const interval = Rx.Observable.interval(5000).timeInterval().take(3)
```

Comment:
Check out the most recent PR for why I wanted to use the Rx pattern, setting up the interval event generator is just that wordy because the library seems to like simple combinators.